mock_controller_driver/README.md

```diff
@@ -0,0 +1,3 @@
+# mock_controller_driver
+
+Mock controller driver replicates the controller driver behavior in for local deployment.
```

Comment:
typo:
"Mock controller driver replicates the controller driver behavior (for example returning engaged status) in local testing deployment."

Reply:
fixed